### DISCUSSION TITLE
Project Column moved webhook

### DISCUSSION THREAD

**Discussion by t3qyo** [community/community#11713] [Projects and Issues]
Hi, I'm a user of GitHub Project.
I'm appreciate to the nice products.
I want to be notified by webhooks of project's column moving action, like from `todo` column to `inprogress` column.
Is there any way to do it?

**Comment by daguar** [comment]
Also looking for this!

**Comment by rileybroughten** [comment]
We've just added the previous and current field value directly in the webhook event when an item is edited, check out this week's [changelog](https://github.blog/changelog/2024-06-27-github-issues-projects-graphql-and-webhook-support-for-project-status-updates-and-more/) for more details 🚀
